docs/source/1.0/spec/aws/aws-cloudformation.rst

```diff
@@ -0,0 +1,672 @@
+=========================
+AWS CloudFormation traits
+=========================
+
+CloudFormation traits are used to describe Smithy resources and their
+components as they would be converted to `CloudFormation Resource Schemas`_.
+
+.. contents:: Table of contents
```

Comment:
After reading this, I think this needs a high-level overview of what these traits do, how they work, and why they're useful. There should be links to the CFN specs when possible too for further reading.